src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -529,15 +521,6 @@ public boolean equals(Object obj) {
 
         // state check
         ModelManager other = (ModelManager) obj;
-        return restOrRant.equals(other.restOrRant)
-                && userPrefs.equals(other.userPrefs)
-                && filteredOrderItems.equals(other.filteredOrderItems)
-                && Objects.equals(selectedOrderItem.get(), other.selectedOrderItem.get())
-                && filteredMenuItems.equals(other.filteredMenuItems)
-                && Objects.equals(selectedMenuItem.get(), other.selectedMenuItem.get())
-                && filteredTableList.equals(other.filteredTableList)
-                && Objects.equals(selectedTable.get(), other.selectedTable.get())
-                && filteredBillList.equals(other.filteredBillList)
-                && Objects.equals(selectedBill.get(), other.selectedBill.get());
+        return restOrRant.equals(other.restOrRant) && userPrefs.equals(other.userPrefs) && filteredOrderItems.equals(other.filteredOrderItems) && Objects.equals(selectedOrderItem.get(), other.selectedOrderItem.get()) && filteredMenuItems.equals(other.filteredMenuItems) && Objects.equals(selectedMenuItem.get(), other.selectedMenuItem.get()) && filteredTableList.equals(other.filteredTableList) && Objects.equals(selectedTable.get(), other.selectedTable.get()) && filteredBillList.equals(other.filteredBillList) && Objects.equals(selectedBill.get(), other.selectedBill.get());
```

Comment:
The code here is compressed into a single line which will result in checkstyle errors